doc/source/whatsnew/v0.20.0.txt

```diff
@@ -88,6 +88,46 @@ support for bz2 compression in the python 2 c-engine improved (:issue:`14874`).
    df = pd.read_table(url, compression='bz2')  # explicitly specify compression
    df.head(2)
 
+.. _whatsnew_0200.enhancements.uint64_support:
+
+Pandas has significantly improved support for operations involving unsigned,
+or purely non-negative, integers. Previously, handling these integers would
+result in improper rounding or data-type casting, leading to incorrect results.
+One notable place where this improved was in ``DataFrame`` creation (:issue:`14917`):
+
```

Comment:
I don't think you need this example. mainly wanted to put the issues together (top section is good though).